### DISCUSSION TITLE
Google Classroom Link Not Working?

### DISCUSSION THREAD

**Discussion by aliciasle** [cooperrc/computational-mechanics#138] [Q&A]
The link to join the Google Classroom that was sent in via email is bringing be to a page that says "Class Not Found".
There is an option to join a class via class code.  Maybe that would work better.  Is there a class code?

**Comment by cooperrc** [comment]
Try this one, 

https://classroom.google.com/c/NTc2NDIzMTY2Njgz?cjc=jgcmvzk

Not sure what went wrong in the email...
